### PR TITLE
HW-346: Files tab page design

### PR DESCRIPTION
**Pull request by mukeshcompucorp**
**Before**

files-tab-issue

**After**

resolved-files-tab-issue

### CODE REVIEW

File: css/activityFeed.css [hunk 1]
```diff
@@ -39,13 +39,20 @@
 }
 
 #bootstrap-theme .act-feed-list-group {
-  width: 100%;
+  margin: 0 auto;
+  width: 80%;
```
Comment (colemanw) [inline css/activityFeed.css]: Have you checked to make sure this doesn't inadvertently affect the "Activities" tab?

File: css/activityFeed.css [hunk 2]
```diff
@@ -179,37 +187,97 @@
   background-color: #e6807f;
 }
 
+#bootstrap-theme .act-feed-list-group .list-group-item:first-child .act-date {
+  border-top: 0;
+  padding-top: 10px;
+}
```
Comment (colemanw) [inline css/activityFeed.css]: Have you checked to make sure this doesn't inadvertently affect the "Activities" tab?

File: css/activityFeed.css (hunk 1 above)
Comment (mukeshcompucorp) [inline css/activityFeed.css]: Fixed.

File: css/activityFeed.css (hunk 2 above)
Comment (mukeshcompucorp) [inline css/activityFeed.css]: Fixed.

**Pull request merged**